config/locales/ja.yml

```diff
@@ -21,11 +21,11 @@ ja:
   locale_name: 日本語
   none: なし
   not_found: 見つかりませんでした
-  forbidden:
-  api_gem_not_found:
+  forbidden: 認可が拒否されました
```

Comment:
How about this?

```diff
- 認可が拒否されました
+ 禁止されています
```

Reply:
Applied in 05dbb319fd2cea4569cd6f96881cab31d5b67c58